core/src/main/java/org/apache/iceberg/avro/AvroSchemaUtil.java

```diff
@@ -47,20 +47,23 @@ private AvroSchemaUtil() {}
   public static final String VALUE_ID_PROP = "value-id";
   public static final String ELEMENT_ID_PROP = "element-id";
   public static final String ADJUST_TO_UTC_PROP = "adjust-to-utc";
+  public static final String UNION_SCHEMA_TO_RECORD = "union-schema-to-record";
 
   private static final Schema NULL = Schema.create(Schema.Type.NULL);
   private static final Schema.Type MAP = Schema.Type.MAP;
   private static final Schema.Type ARRAY = Schema.Type.ARRAY;
   private static final Schema.Type UNION = Schema.Type.UNION;
   private static final Schema.Type RECORD = Schema.Type.RECORD;
 
-  public static Schema convert(org.apache.iceberg.Schema schema,
-                               String tableName) {
+  public static Schema convert(
+      org.apache.iceberg.Schema schema,
+      String tableName) {
```

Comment:
Minor: please revert non-functional changes like this one. That makes it easier to review and less likely to conflict with other commits.

Reply:
ack I did not do it intentionally , formatting style from .baseline applied these changes.